### PR TITLE
Smartereye

### PR DESCRIPTION
**Pull request by lishuanghua**
This is a hard device driver which is added to apollo as a third party perception device.

### CODE REVIEW

File: modules/drivers/smartereye/compress_component.cc [hunk 1]
```diff
@@ -0,0 +1,84 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
```
Comment (jinghaomiao) [inline modules/drivers/smartereye/compress_component.cc:2]: 2020 and so on and so forth.

**Comment by jinghaomiao** [comment]
@lishuanghua please split your pull requests with multiple commits for easy code review and management purpose.

**Comment by lx18233184051** [comment]
[06:01:55][Step 2/6] //modules/drivers/smartereye:smartereye_component_lib_cpplint            FAILED 
[06:01:55][Step 2/6] //modules/drivers/smartereye:smartereye_device_cpplint                   FAILED in 0.3s
[06:01:55][Step 2/6] //modules/drivers/smartereye:smartereye_handler_cpplint                  FAILED in 0.4s

**Comment by Capri2014** [comment]
Also looks like they are depending on external libs. (*.so)? Please correct me if I am wrong. If so, we generally prefer  to host  those as  third party libs and manage them with bazel or third party libs installers, in stead of directly check those in. @lishuanghua

**Pull request closed without merging**